examples/README.md

```diff
@@ -165,6 +165,7 @@ Example | Description
 [Animated Fox](../examples/animation/animated_fox.rs) | Plays an animation from a skinned glTF
 [Animated Transform](../examples/animation/animated_transform.rs) | Create and play an animation defined by code that operates on the `Transform` component
 [Animation Graph](../examples/animation/animation_graph.rs) | Blends multiple animations together with a graph
+[Color animation](../examples/animation/color_animation.rs) | Bezier curve example showing squares changing colors using different color spaces
```

Comment:
This should be focused on "what does the user hope to learn", not "what does the example do to teach the user". Something like "Demonstrates how to animate colors using linear blending and splines" would be much more helpful to the reader :)